### ISSUE TITLE
Add datetime to commit flags

### ISSUE THREAD

**Issue by lachmanfrantisek** [packit-service/ogr#344]
- [x] add the `created` and `edited` properties to the commit flag classes
  - [x] abstract class
  - [x] github:
    - https://pygithub.readthedocs.io/en/latest/github_objects/Commit.html#github.Commit.Commit.get_statuses
  - [ ] gitlab:
    - https://python-gitlab.readthedocs.io/en/stable/gl_objects/commits.html#commit-status
    - https://docs.gitlab.com/ce/api/commits.html
  - pagure:
    -  https://src.fedoraproject.org/api/0/ -> `Flags for a commit`
- [x] tests for all implementations

**Comment by TomasJani** [comment]
I would like to work on this if possible (as part of RHOSC).

**Comment by lachmanfrantisek** [comment]
@TomasJani nice!

Feel free to ask if you need some help.

**Comment by mfocko** [comment]
Closed by #374 

@lachmanfrantisek can create follow-up issue for the `edited` on GitLab, but I don't think it's the API problem

**Comment by lachmanfrantisek** [comment]
Thanks. Yes please, create a follow-up for the missing parts. (Also issues blocked by API are useful and we shout create an issue in the upstream project for each of them.)
